scss/_navbar-off-canvas.scss

```diff
@@ -0,0 +1,295 @@
+// Overrides for the Navbar component for off-canvas menu.
+.offcanvas-toggle {
+  display: none;
+}
+@keyframes fadein {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: .75;
+  }
+}
+// >> .navbar-offcanvas
+.menu-backdrop {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1040;
+  background-color: rgb(64, 54, 53);
+  opacity: .75;
+}
+.menu-backdrop.show {
+  cursor: pointer;
+  animation: .3s linear fadein;
+}
+
+.navbar-off-canvas {
```

Comment:
I also have this entire block nested inside a media breakpoint so that these styles are only being applied at the mobile size. I've attached a .txt file with the most recent SCSS I have.
[_navbar-offcanvas.txt](https://github.com/az-digital/arizona-bootstrap/files/5211036/_navbar-offcanvas.txt)